CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Support for RPC v0.6.0-rc1 via the `/rpc/v0_6` endpoint. Note that this does not include the `/rpc/v0.6` endpoint as the underscore is now the standard across node implementations.
 
+- Support for Testnet and Integration chains (running against Ethereum Sepolia testnet).
+
 ### Changed
 
 - Default RPC version is now v0.5 (was v0.4). This can be manually configured to any version.
```

Comment:
```suggestion
```suggestion
- Support for RPC v0.6.0-rc1 via the `/rpc/v0_6` endpoint. Note that this does not include the `/rpc/v0.6` endpoint as the underscore is now the standard across node implementations.
- Support for Sepolia testnet via `--network testnet/sepolia`
- Support for Sepolia integration via `--network integration/sepolia`

### Changed

- Default RPC version is now v0.5 (was v0.4). This can be manually configured to any version.
- Goerli testnet network selection is now `--network testnet/goerli`, removed `--network testnet`
- Goerli integration network selection is now `--network integration/goerli`, removed `--network testnet`
```